File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L6/06/fungsi.kt
package IR

import java.util.*

object fungsi {
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")

        val sc = Scanner(System.`in`)
        val input = sc.nextInt() //input integer
        terbalik(input) //memanggil fungsi reverse
    }

    fun terbalik(angka: Int) {
        var angka = angka
        val arr: MutableList<*> = ArrayList<Any?>()
        run {
            var i = 0
            while (i <= angka) {
                //looping sampai angka habis
                var sisa = angka % 10 //membagi bilangan tersebut dengan 0 sampai tidak ada hasil bagi
                arr.add(sisa)
                angka = angka / 10
                i = 0
                i++
            }
        }
        for (i in arr.indices) {
            print(arr[i])
        }

        println()
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L5/06/fungsi.py
import sys

def terbalik(angka):
    while angka > 0:
        sisa = angka % 10
        print(sisa, end='')
        angka //= 10
    print()

if __name__ == "__main__":
    print("Enter an integer: ", end='')
    input_num = int(input())
    terbalik(input_num)

